Dockerfile

```
@@ -0,0 +1,32 @@
+FROM php:7.2-fpm-alpine
+
+RUN apk update && apk upgrade \
+    && apk add --no-cache \
+        freetype \
+        libpng \
+        libjpeg-turbo \
+        freetype-dev \
+        libpng-dev \
+        jpeg-dev \
+        libjpeg \
+        libjpeg-turbo-dev \
```

Wouldn't this solve the iconv problem instead of going the full Debian way?
```suggestion
        libjpeg-turbo-dev \
        gnu-libiconv \
```

---

Yep, that was my "I know why this one errs", mentioning debian was about something different, In my defence, I did say "this wall of text will be cleaned up into something more cohesive before it is taken out of draft" 🤣  I Should have had another coffee before pushing that up.

Have you tested this MR, does the workflow Work fine for you?

---

I've now installed a Docker environment on my Windows laptop and have been able to test it. I had to tweak a few things to make it work, but other than that, it looks really good.

Here's what I had to do to make it run under Windows (without make...):

1. Install Docker Desktop
2. Activate the Shared Drives in the settings of Docker Desktop
3. Create a `.env` file based on the `.env.example` file (BTW, the `.env` file should be added to `.gitignore`)
4. Don't run `docker pull --parallel`, as it doesn't work
5. Run `docker build --rm --no-cache --pull -f Dockerfile -t phpdvdprofiler:latest .`
6. Run `docker-compose up`
7. Profit!

Thanks a lot!